tools/cockpit.spec

```diff
@@ -53,12 +53,9 @@ Version:        0
 Release:        1%{?dist}
 Source0:        https://github.com/cockpit-project/cockpit/releases/download/%{version}/cockpit-%{version}.tar.xz
 
-%if 0%{?fedora} >= 38 || 0%{?rhel} >= 9
-%define cockpit_enable_python 1
-%endif
-
-%if !%{defined cockpit_enable_python}
-%define cockpit_enable_python 0
+# Don't change the bridge in the stable release; also, the old Python breaks some features
```

Comment:
RHEL 9 is also stable, isn't it?

And I think it's not the old Python which breaks features, but rather the old SSH.  And I think we worked around that, didn't we...?

AFAIK there's nothing that stops us from running on RHEL 8, except fear itself :)

Reply:
Oh yes, there are still 18 todoPybridgeRHEL8 failures, mostly because of the broken ferny/ssh indeed. So we couldn't switch RHEL 8 right now even if we dared to. For a RHEL 8 beiboot target we just accept that you can't ssh further from that (we don't want to support transitive SSH anyway).

I clarified the comment.